src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -5053,7 +5061,7 @@ R"HTML(
        {
           int oct_offset = 1;
              oct_offset = Surge::Storage::getUserDefaultValue(&(this->synth->storage), "middleC", 1);
-          char notename[16];
+          char notename[8];
```

Comment:
Oh sorry this 16 -> 8 is gonna break Ubuntu 20 also and looks like another merge (it was in the same diff as the Linux opt above). As soon as Ubuntu 20 is in azure I will activate it